src/ExtendableSpeciesCard.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import EbiSpeciesIcon from 'react-ebi-species'
+
+const MAX = 5
+
+class ExtendableSpeciesCard extends React.Component {
+
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isHidden: this.props.content ? this.props.content.length >= MAX : false
+    }
+    this.onClick = this.onClick.bind(this)
+  }
+ 
+  onClick() {
+    this.setState({
+      isHidden: !this.state.isHidden
+    })
+  }
+    
+  render() {
+    const {iconSrc, iconDescription, content} = this.props
+    
+    let visibleContent = (max, content) => {
+      return content.slice(0, max).map((item) => {
+        return item.url ?
+                <li className={`url`} style={{marginBottom: `0.3rem`}} key={item.text}><a href={item.url} key={item.text}>{item.text}</a></li> :
+                <li className={`text`} style={{marginBottom: `0.3rem`}} key={item.text}>{item.text}</li>
+      })
+    }
+
+    return (
+      <div className={`column column-block text-center combo card`} style={{marginBottom:0, paddingBottom: `25px`}}>
+        {
+          iconDescription && <h5 className="species-name">{iconDescription}</h5>
+        }
+
+        <span className={`species-icon`} style={{fontSize: `800%`}}>
+          <EbiSpeciesIcon species={iconSrc}/>
+        </span>
+        {
+          content &&
+            <ul className={`content`} style={{listStyle:`none`, paddingLeft:`0`, marginLeft:`0`}}>
+              {
+                this.state.isHidden ?
+                  <div>
+                    {visibleContent(MAX, content)}
+                    <button className="button small show_button" onClick={this.onClick}> See more…</button>
+                  </div>
+                  :
+                  <div>
+                    {visibleContent(content.length, content)}
+                    {
+                      content.length >= MAX ?
+                        <button className="button small hide_button" onClick={this.onClick}> Hide…</button> :
+                        null
+                    }
+                  </div>
+              }
```

Comment:
This can be even simpler. Let’s say your `visibleContent` function is something like this:
```
const visibleContent = content.map(item) => {
  return item.url ?
    <li className={`url`} style={{marginBottom: `0.3rem`}} key={item.text}><a href={item.url} key={item.text}>{item.text}</a></li> :
    <li className={`text`} style={{marginBottom: `0.3rem`}} key={item.text}>{item.text}</li>
})
```

Then  you can simply do (notice also that here the `show_button` and `hide_button` classes serve no purpose):
```
<div>
  {visibleContent.slice(0, MAX)}
  <button ...>{this.state.hidden ? `Show all` : `Show less`}</button>
</div>
```

The important point here is that `slice` gives you back the array unmodified if you specify a longer portion:
```
> [1,2,3].slice(0,5)
[ 1, 2, 3 ]
```